### ISSUE TITLE
<Smarttech247> Job title mismatch/incomplete

### ISSUE THREAD

**Issue by GeorgePara** [peviitor-ro/issues#170]
ENVIRONMENT: PROD
 url: https://firme.peviitor.ro/
 browser: Chrome

STEPS TO REPRODUCE:
 1)open url in the browser
 2)type Smarttech247 in the search bar
 3)check the job title for "Product Marketing Manager"
 4)click on "Vezi Postul->"
 5)check the job title

EXPECTED RESULTS:
 the job title should be listed as "Product Marketing Manager – Cybersecurity"

ACTUAL RESULTS:
 the job title is listed as "Product Marketing Manager"

![smarttech247JobTitleMismatch1](https://github.com/peviitor-ro/issues/assets/135150121/701fe243-8820-4ea0-a43b-72c733093b57)
![smarttech247JobTitleMismatch2](https://github.com/peviitor-ro/issues/assets/135150121/451467fc-3494-4e28-8442-003a82091381)

**Comment by stefaniavlad** [comment]
Searched on their careers page for Product Marketing Manager - Cybersecurity role and on the main career page the title looks like the one from peviitor.ro.
![image](https://github.com/peviitor-ro/issues/assets/135824216/735bbd85-a109-4f88-9566-6a638cf86321)


But if I open the job, indeed the job title looks different.
![image](https://github.com/peviitor-ro/issues/assets/135824216/d4839968-e718-4f91-a24d-d3a8b9295ffa)

I think it's a bug from their career page, as we do not enter each job page one by one and take the title from there, we just take the title and the link from their main career page.

Also, I inspected a little bit the html of the page:
![image](https://github.com/peviitor-ro/issues/assets/135824216/073ed2f3-cec7-4f99-a0e4-c47eb1a8be1f)


I couldn't find any "- Cybersecurity" in this div section, but on their job page the title appears to be somewhere else:
![image](https://github.com/peviitor-ro/issues/assets/135824216/85f141f0-91db-468b-ad8d-15de2d89625f)

In conclusion, I think it's a bug from their side, not from us.
